src/main/java/com/netgrif/application/engine/configuration/SecurityConfiguration.java

```diff
@@ -181,4 +187,8 @@ private PublicAuthenticationFilter createPublicAuthenticationFilter() throws Exc
                 this.userService
         );
     }
+
+    private SecurityContextFilter createSecurityContextFilter() throws Exception {
```

Comment:
is it possible to specify a more concrete `Exception` class here?

Reply:
No, because there is a spring class used in that function that throws Exception.